calblueprint/vouchers4veggies
# [wip][scanning] Implementation for transaction review screen

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Set up review transactions screen following voucher scanning flow.


## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Configure frontend for screen
 - [ ] [TODO] Link navigation to appropriate screens when finalized (back button, submission)

## How can the reviewer test your code? 👩‍💻
_example: run `npm start` and click through Navigation bar_ 

`expo start` then scan a voucher, push 'Review & Submit' to see screen

**Transferring scanned voucher data + adding to backend will be a separate sprint**

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 

Responsiveness of ScrollView is not the greatest, leaves some whitespace below submit button on larger devices. 


## Resources 📔
_link online resources and related PRs_ 

🥦 CC: @sauhardjain
